net-irc/irssi/irssi-0.8.19_git.recipe

```diff
@@ -0,0 +1,68 @@
+LICENSE="GNU GPL v2"
+COPYRIGHT="1999-2015 Timo Sirainen"
```

Comment:
Please follow the order outlined in https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#ordering